CONTRIBUTING.md

```diff
@@ -152,6 +153,12 @@ See
 on how to get started, or jump right into translating at
 [crowdin.com/project/electron](https://crowdin.com/project/electron).
 
+### Announcement Banner
+
+The top-level banner for showing the information (for example new Electron version, important blog post).
+
+For show or update the announcement banner modify the [message](https://github.com/electron/electronjs.org/blob/master/data/locale.yml#L220) and update the show [time](https://github.com/electron/electronjs.org/blob/master/routes/home.js#L5).
```

Comment:
I'm generally hesitant to add code links to `master`, mostly because line numbers change over time. Can we either have permalinks as examples of previous uses, and/or be more specific in this document as to what to change?

Minor grammatical changes:

```suggestion
To show or update the announcement banner, modify the [message](https://github.com/electron/electronjs.org/blob/master/data/locale.yml#L220) and update the show [time](https://github.com/electron/electronjs.org/blob/master/routes/home.js#L5).
```